examples/socketmode/events.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slackevents"
+)
+
+// You more than likely want your "Bot User OAuth Access Token" which starts with "xoxb-"
+var api = slack.New("TOKEN")
+
+func main() {
+	api := slack.New(
+		"YOUR TOKEN HERE",
+		slack.OptionDebug(true),
+		slack.OptionLog(log.New(os.Stdout, "slack-bot: ", log.Lshortfile|log.LstdFlags)),
+	)
+
+	client := api.NewSocketModeClient()
+	go client.ManageConnection()
+
+	for evt := range client.IncomingEvents {
+		eventsAPIEvent, ok := evt.Data.(slackevents.EventsAPIEvent)
```

Comment:
How does handing of "interactive"/"slash_commands" type envelopes look like?

Reply:
They look like https://github.com/mumoshu/slack/blob/0be002ac0a943e28b8651abe5c65a0f00097c465/examples/socketmode/socketmode.go#L79-L96 now